common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala

```diff
@@ -9,24 +9,18 @@ import Scalaz._
 import com.gu.mediaservice.model.{StaffPhotographer, ContractPhotographer, Photographer}
 
 object PhotographersList {
-  type Store = Map[String, String]
   type CreditBylineMap = Map[String, List[String]]
 
-  import MetadataConfig.{ staffPhotographers, contractedPhotographers }
+  import MetadataConfig.{ Store, staffPhotographers, contractedPhotographers }
 
   def creditBylineMap(store: Store): CreditBylineMap = store
-      .groupBy{ case (photographer, publication) => publication }
-      .map{ case (publication, photographers) => publication -> photographers.keys.toList.sortWith(_.toLowerCase < _.toLowerCase) }
+      .groupBy{ case (_, publication) => publication }
+      .map{ case (publication, photographers) => publication -> photographers.map(_._1).sortWith(_.toLowerCase < _.toLowerCase) }
 
-  def creditBylineMap(stores: List[Store]): CreditBylineMap =
-    stores.map(creditBylineMap).reduceLeft(_ |+| _)
-
-  def list(store: Store) = store.keys.toList.sortWith(_.toLowerCase < _.toLowerCase)
-
-  def getPublication(store: Store, name: String): Option[String] = store.get(name)
```

Comment:
the new shape of store means this would either need a `.toMap` or to instead use `.find` like the `caseInsensitiveLookup` below, but it's never used so removed it instead